lib/index.js

```diff
@@ -8,30 +8,40 @@
  */
 
 const path = require('path');
-const engine = require('unified-engine-atom');
+
+let engine;
+let processor;
 
 /**
  * Linter-markdown.
  *
  * @return {LinterConfiguration}
  */
 function linter() {
+  const config = atom.config.get('linter-markdown');
```

Comment:
Why not observe `linter-markdown.detectIgnore` in the `activate()` method? There isn't a reason to do all the parsing required to grab the entire config here.